packages/react/src/components/DetailsList/DetailsColumn.base.tsx

```diff
@@ -232,6 +237,17 @@ export class DetailsColumnBase extends React.Component<IDetailsColumnProps> {
     }
   }
 
+  private _onRenderFilterIcon = (classNames: IProcessedStyleSet<IDetailsColumnStyles>) => (
+    props: IDetailsColumnFilterIconProps,
+  ): JSX.Element => {
+    const { columnProps, ...iconProps } = props;
+    const IconComponent = columnProps?.useFastIcons ? FontIcon : Icon;
+
+    return (
+      <IconComponent iconName="ChevronDown" aria-hidden={true} className={classNames.filterChevron} {...iconProps} />
```

Comment:
These values should come from `props`. That's how a host component can easily override them without having to invoke `IconComponent` itself.

Reply:
@ThomasMichon Updated the PR with the default props